test/hummingbot/data_feed/liquidations_feed/test_liquidations_factory.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+
+from hummingbot.data_feed.liquidations_feed.binance import BinancePerpetualLiquidations
+from hummingbot.data_feed.liquidations_feed.liquidations_factory import LiquidationsConfig, LiquidationsFactory
+
+
+class TestCandlesFactory(unittest.TestCase):
```

Comment:
TestLiquidationsFactory*

Reply:
just done, this one I forgot as it seems